expression/builtin_math_vec_test.go

```diff
@@ -119,6 +119,12 @@ var vecBuiltinMathCases = map[string][]vecExprBenchCase{
 	ast.CRC32: {
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETString}},
 	},
+	ast.Conv: {
+		{
+			retEvalType:   types.ETString,
+			childrenTypes: []types.EvalType{types.ETString, types.ETInt, types.ETInt},
```

Comment:
We may need to limit the rune in `arg[0]` because the range of each rune should be `[A-Z]|[0-9]`.
We may need to limit the bound of the `arg[1]` and `arg[2]` because the valid number is 2 to 36, maybe a range of  `[-1, 40]` is ok.